build_resolvers/test/resolver_test.dart

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
+//
+// @dart=2.9
```

Comment:
This is blocked on build_test right? Plan is to just come back and opt these in after landing both prs?

Reply:
I can migrate this test in the second PR, looks like I just missed it.

Reply:
Actually I already did: https://github.com/dart-lang/build/pull/3018/commits/273e80bdf687551880574fcccc5335cfdfbbbe5f